### DISCUSSION TITLE
스프링 프로파일 관리

### DISCUSSION THREAD

**Discussion by janeljs** [Sc0up/scoup-backend#52] [New Feature]
dev, prod 환경에 따른 OAuth, DB, 로깅 레벨 설정에 대해 논의해봅시다.

**Comment by Dae-Hwa** [comment]
인수 테스트 진행 시 locale 설정 고려 필요
![image](https://user-images.githubusercontent.com/24666330/129223559-632dbed7-4c25-442b-bf3b-edf9b2775f30.png)

**Reply by Dae-Hwa** [reply]
테스트 프로파일에만 넣는 걸로 결정됨
